### PR TITLE
0.8.0-preview

### PR DESCRIPTION
**Pull request by oblm**
**Contributions**: We are not currently taking public contributions - see our [contributions](CONTRIBUTING.md) policy. However, we are accepting issues and we do want your [feedback](../README.md#give-us-feedback).

-------

#### Description
This PR releases `0.8.0-preview` to `preview`.

#### Release note
See CHANGELOG.md.

#### Tests
The [release process](https://github.com/spatialos/UnrealGDK/blob/master/SpatialGDK/Extras/internal-documentation/release-process.md) testing was run three times: twice on engine 4.23 & once on engine 4.22.

#### Documentation
See CHANGELOG.md and `docs-preview`.

### CODE REVIEW

**Comment by joshuahuburn** [comment]
Due to merge conflicts have merged preview into the 0.8.0-preview-rc. This will now need a cloud deploy to verify.

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,7 +4,53 @@ All notable changes to the SpatialOS Game Development Kit for Unreal will be doc
...
+- Replicating a static subobject after it has been deleted on a client no longer results in client attaching a new dynamic subobject.
+- Fixed a bug that caused entity pool reservations to cease after a request times out.
+- Running `BuildWorker.bat` for `SimulatedPlayer` no longer fails if the project path has a space in it.
+- Fixed a crash when starting PIE with out-of-date schema.
+
+### Internal:
+Features listed in the internal section are not ready to use but, in the spirit of open development, we detail every change we make to the GDK.
+- We've added a partial loadbalancing framework. When this is completed in a future release, you will be able to control loadbalancing using server-workers.
+
+## [`0.7.1-preview`] - 2019-12-06
```
Comment (m-samiec) [inline CHANGELOG.md]: This section is duped

Comment (oblm) [reply CHANGELOG.md]: https://github.com/spatialos/UnrealGDK/pull/1619

File: CHANGELOG.md [hunk 2]
```diff
@@ -43,12 +80,13 @@ Features listed in the internal section are not ready to use but, in the spirit
 
 ### Features:
 - The GDK now uses SpatialOS `14.1.0`.
+- Added in-editor support for exposing a local runtime at a particular IP address. This offers the same functionality as the `--runtime_ip` option in the SpatialOS CLI.
```
Comment (joshuahuburn) [inline CHANGELOG.md]: Hmmmm, these seem in the wrong place actually, they shouldn't be going into 0.7.0-preview should they?

File: CHANGELOG.md [hunk 3]
```diff
@@ -74,6 +113,7 @@ Features listed in the internal section are not ready to use but, in the spirit
 - Muticast RPCs that are sent shortly after an actor is created are now correctly processed by all clients.
 - When replicating an actor, the owner's Spatial position will no longer be used if it isn't replicated.
 - Fixed a crash upon checking out an actor with a deleted static subobject.
+- Fixed an issue where launching a cloud deployment with an invalid assembly name or deployment name wouldn't show a helpful error message
```
Comment (joshuahuburn) [inline CHANGELOG.md]:
```suggestion
- Fixed an issue where launching a cloud deployment with an invalid assembly name or deployment name wouldn't show a helpful error message.
```

**Pull request merged**